rskj-core/src/main/java/org/ethereum/vm/PrecompiledContracts.java

```diff
@@ -63,7 +63,15 @@
     public static final RskAddress REMASC_ADDR = new RskAddress(REMASC_ADDR_STR);
     public static final RskAddress SAMPLE_ADDR = new RskAddress(SAMPLE_ADDR_STR);
 
-    private static final String RSK_NATIVECONTRACT_REQUIREDPREFIX = "000000000000000000000000";
```

Comment:
Could you elaborate on why is safe to remove this prefix? couldn't this imply an unplanned hard fork?

Reply:
The prefix is still tested. What is not used anymore is the string variable.
Comparisons with DataWords use always the full 32 byte value.